puppet/modules/jenkins_job_builder/files/centos.org/ansible/fetch_debug_files.yml

```diff
@@ -0,0 +1,14 @@
+---
+- hosts: any
```

Comment:
Whats the difference between any and all?

Reply:
Just a typo on my part!